04/styles/mobile.css

```diff
@@ -0,0 +1,72 @@
+ul {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+    color: black;
+
+}
+
+li {
+    outline: 1px solid black;
+    text-align: center;
+}
+
+.second-menu  {
+    display: block;
+}
+
+.menu {
+    display: flex;
+    justify-content: right;
+}
+
+ul {
+    padding: 0;
+    margin: 0;
+    display: none;
+}
+
+nav input:checked + ul {
+    display: block;
+}
+
+.hamburger {
+    width: 40px;
+    height: 35px;
+    border: 0;
+    border-top: 5px solid black;
+    background: silver;
+    position: relative;
+    background: transparent;
+    transition: 0.3s transform linear;
+    
+}
+
+.hamburger::before, .hamburger::after {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    border-top: 5px solid black;
+    transform: translateY(10px)
+}
+
+.hamburger::after {
+    transform: translateY(25px);
+    transition: 0.3s transform linear;
+}
+
+.hamburger:focus + ul {
+    display: block;
+}
+
+.menu:hover + ul {
+    display: block;
+}
+
+.second-menu li {
+    text-align: right;
+}
```

Comment:
Na wersji mobilnej podmenu nie zbyt dobrze wygląda
![image](https://user-images.githubusercontent.com/55095257/147806877-5453f435-846a-4f51-afd1-ff33af4a1718.png)

Reply:
Wprowadziłem zmiany w kodzie i pozycje z "second-menu" są teraz odsunięte od lewej krawędzi i wycentrowane, jednak zrobiłem to kosztem usunięcia zagnieżdżenia, czyli drugi znacznik <ul> nie jest już zagnieżdżony w <li>oferta</li>, a chyba powinien być.. 


![Zrzut ekranu 2022-01-01 o 15 29 28](https://user-images.githubusercontent.com/80978376/147852881-17a025df-e041-4b5f-9a51-e461f9b1bd2d.png)

Reply:
Jak już pisałem, zagnieżdżenie powinno być. Odsunąć od lewej krawędzi możesz przez użycie `padding-left` dla konkretnej klasy :)